website/src/components/CastPlayer/styles.module.css

```
@@ -0,0 +1,277 @@
+}
+
+.screen {
+  flex: 1 1 auto;
+  min-height: 0;
+  height: auto;
+  margin: 0;
+  padding: 3.25rem 2.5rem 5.5rem;
+  overflow: auto;
+  background: var(--atmos-terminal-bg);
+  color: var(--atmos-terminal-text);
+  font-family: 'SFMono-Regular', Menlo, Monaco, Consolas, 'Liberation Mono', 'Courier New', monospace;
```

Stylelint (17.14.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)